devops/scripts/load_and_validate_env.sh

```diff
@@ -85,4 +85,13 @@ else
     export TRE_URL
 fi
 
+# if local debugging is configured, then set vars required by ~/.porter/config.yaml
+if [ -f "$DIR/../../core/private.env" ]; then
+  # shellcheck disable=SC1091
+  source "$DIR/../../core/private.env"
+  # shellcheck disable=SC2153
+  KEY_VAULT_URL=$KEYVAULT_URI
+  export KEY_VAULT_URL
```

Comment:
What is special about this one that it needs this special procedure?

Reply:
So to debug the resource processer the porter `config.yaml` is present. This requires the environment variable `KEY_VAULT_URL` to be set so the secret can be retrieved, this is set in the resource processor `config.py` code.

Without this env variable set the commands in the makefile that call porter fail as it can't resolve the keyvault to access the porter state. That make sense? If you have another way of doing this, do say so...